### PR TITLE
refactor: add Metadata and refactor components

### PR DESCRIPTION
**Pull request by sauldeleon**
Add Metadata component
Refactored components in its proper folders

### CODE REVIEW

File: packages/blog/src/api/category/categories.mock.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        updatedAt: '2022-04-15T20:07:30.237Z',
+        code: 'ENG',
+        main: true,
+      },
+    },
+    {
+      id: 4,
+      attributes: {
+        name: 'News',
+        description: 'Current news arround the World',
```
Comment (paulaguijarro) [inline packages/blog/src/api/category/categories.mock.ts:40]:
```suggestion
        description: 'Current news around the world',
```

File: packages/blog/src/components/home/HomePage/HomePage.tsx [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
-      categories={['travel', 'lifestyle']}
-      title="Vacaciones en el Algarve. Una experiencia inolvidable."
-      subtitle="Un viaje por el Algarve y sus playas, descubriendo la paz del Atlántico. Sus gentes y su gastronomía de cerca."
-    />
+    <>
+      <Metadata name={t('homePage.title')} description={t('homePage.description')} />
+      <HeaderImage
+        imgSrc="https://images.unsplash.com/photo-1506477331477-33d5d8b3dc85?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=868&q=80"
+        date={new Date()}
+        categories={['travel', 'lifestyle']}
```
Comment (paulaguijarro) [inline packages/blog/src/components/home/HomePage/HomePage.tsx:13]: Mixing English and Spanish

**Review by paulaguijarro — APPROVED** [review]

File: packages/blog/src/api/category/categories.mock.ts (hunk 1 above)
Comment (sauldeleon) [reply packages/blog/src/api/category/categories.mock.ts]: Fixed. Thanks!

File: packages/blog/src/components/home/HomePage/HomePage.tsx (hunk 2 above)
Comment (sauldeleon) [reply packages/blog/src/components/home/HomePage/HomePage.tsx:13]: This will be fixed when adding dynamic categories within a post!

**Pull request merged**